x/attribute/types/keys.go

```diff
@@ -94,6 +113,13 @@ func GetNameKeyBytes(name string) []byte {
 	return hash[:]
 }
 
+// GetAttributeExpireTimePrefix returns a prefix for expired time [AttributeExpirationKeyPrefix][epoch]
+func GetAttributeExpireTimePrefix(expireTime time.Time) []byte {
+	buf := make([]byte, 8)
```

Comment:
Seems like we usually have a set of constants with these magic numbers listed at the top ... makes is harder to make a mistake and re-use or skip a key unique id number in the current context

Reply:
Ok, I added the prefix to the top of the method.  I think that is what you meant?